contenteditable.js

```diff
@@ -0,0 +1,42 @@
+(function() {
+  var inputList = $$('.param');
+  var hiddenTextList = $$('.hidden-text');
+  function onInput(input, index) {
+    hiddenTextList[index].innerHTML = input.value || 0;
+  }
+  function onKeydown(event, input) {
+    if (event.key === '-' && input.value[0] === '-') {
+      event.preventDefault();
```

Comment:
Prevent adding another minus sign because when there are two minus signs, the value of <input type="number"> becomes an empty string.